ambari-logsearch-logfeeder/src/main/java/org/apache/ambari/logfeeder/conf/CloudStorageDestination.java

```diff
@@ -19,7 +19,7 @@
 package org.apache.ambari.logfeeder.conf;
 
 public enum CloudStorageDestination {
-  HDFS("hdfs"), S3("s3"), GCS("gcs"), ADLS("adls"), NONE("none");
+  HDFS("hdfs"), S3("s3"), GCS("gcs"), ADLS("adls"), WASB("wasb"), NONE("none");
```

Comment:
Do we need to add secure mode enums? (adlss, wasbs) are the ones that I know exist.

Reply:
it is something for logfeeder level, but as we are not supporting core gcs/wasb yet it won't be needed anyway, default-fs is used, if hdfs client will figure out the right filesystem

Reply:
Makes sense.